django/chat/migrations/0005_chat_last_message_date.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 5.1.5 on 2025-02-28 18:35
+
+import django.utils.timezone
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("chat", "0004_alter_message_options"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="chat",
+            name="last_message_date",
+            field=models.DateTimeField(default=django.utils.timezone.now),
```

Comment:
Should set this to the chat.last_accessed in the migration. I will push a commit that does this.